sdk/keyvault/keyvault-certificates/test/mergeAndImport.test.ts

```diff
@@ -54,15 +55,38 @@ describe("Certificates client - merge and import certificates", () => {
     const certificateSecret = await secretClient.getSecret(certificateNames[0]);
     const base64EncodedCertificate = certificateSecret.value!;
```

Comment:
Better to use a dummy certificate created outside KV, and use that to import.

Reply:
Let's not do that as part of this PR. I understand where you come from, but that will add considerable complexity to this PR.